.docker_files/Dockerfile

```
@@ -0,0 +1,30 @@
+FROM odoo:11.0
+MAINTAINER numigi <contact@numigi.com>
+
+USER root
+
+ENV ODOO_DIR /usr/lib/python3/dist-packages
+
+RUN rm -rf ${ODOO_DIR}/odoo
+
+COPY odoo ${ODOO_DIR}/odoo
+COPY addons ${ODOO_DIR}/odoo-addons
+
+RUN mv ${ODOO_DIR}/odoo/addons/* ${ODOO_DIR}/odoo-addons/ && \
+    mv -f ${ODOO_DIR}/odoo-addons/* ${ODOO_DIR}/odoo/addons
+
+# Freeze pip and wheel
+RUN pip3 install pip==9.0.1 wheel==0.30.0
```

mettre à jour les version des packages pip
```suggestion
RUN pip3 install pip==18.1 wheel==0.32.1
```

---

@foutoucour done